source/routing/raptor_api.cpp

```diff
@@ -1452,6 +1449,44 @@ void make_isochrone(navitia::PbCreator& pb_creator,
     }
 }
 
+void make_isochrone_distributed(navitia::PbCreator& pb_creator,
+                                RAPTOR& raptor,
+                                const type::EntryPoint& isochrone_center,
+                                const std::vector<type::EntryPoint>& stop_points,
+                                const uint64_t datetime_timestamp,
+                                bool clockwise,
+                                const type::AccessibiliteParams& accessibilite_params,
+                                std::vector<std::string> forbidden,
```

Comment:
```suggestion
                                const std::vector<std::string>& forbidden,
```
?
Try to do a maximum of const and reference if possible.

Reply:
Yes indeed! I've missed that! Thanks 😉

Reply:
:+1: 
There are some input function without `const ref` input prameters. Takes a ride in the PR :pray:

Reply:
Done everywhere I could! :smile: